backend/cmd/headlamp.go

```diff
@@ -463,20 +471,26 @@ func createHeadlampHandler(config *HeadlampConfig) http.Handler {
 
 		kContext, err := config.kubeConfigStore.GetContext(cluster)
 		if err != nil {
-			log.Printf("Error: failed to get context: %s", err)
+			str := map[string]string{"cluster": cluster}
+			utils.Log(utils.LevelError, str, err, "Error: failed to get context")
+
 			http.NotFound(w, r)
 			return
 		}
 
 		oidcAuthConfig, err := kContext.OidcConfig()
 		if err != nil {
-			log.Printf("Error getting %s cluster oidc config %s", cluster, err)
+			str := map[string]string{"cluster": cluster}
```

Comment:
similar to other comments no need to create the map as a separate variable since it is used only as an argument to the `utils.Log` function.